src/arm64/tests/test-detection.yaml

```diff
@@ -8,8 +8,6 @@ min_bb_per_function: 2
 max_bb_per_function: 2
 avg_mem_accesses: 4
 input_gen_entropy_bits: 9
-
-enable_ssbp_patch: true
```

Comment:
Doesn't ARM have a patch against Speculative Store Bypass (SSBP)?

Reply:
Yes, Arm has the Physical SSBP Barrier instruction ([PSSBB](https://developer.arm.com/documentation/ddi0597/2021-03/Base-Instructions/PSSBB--Physical-Speculative-Store-Bypass-Barrier-?lang=en)) and the "Speculative Store Bypass Safe" register ([SSBS](https://developer.arm.com/documentation/ddi0601/2023-09/AArch64-Registers/SSBS--Speculative-Store-Bypass-Safe)), which toggles their patch.

I removed this field from the YAML because I couldn't find it being used anywhere in the Arm port code. (Last year, we didn't implement it as part of the Hackathon. Unless it was added since then, support for that does not exist in the Arm port.) My intention was to clean up the test YAML, but I'm happy to add this back in if you'd prefer it be left alone until Arm SSBP support can be added in.